src/Nest/Cluster/NodesStats/NodeStats.cs

```diff
@@ -85,6 +92,9 @@ public class ScriptStats
 
 		[DataMember(Name = "compilations")]
 		public long Compilations { get; internal set; }
+
+		[DataMember(Name = "compilation_limit_triggered")]
```

Comment:
Just double checking this always comes back as `0` from the server?

Reply:
The value returned is always `0`? Might be tricky to have a valid greater than `0` returned in tests.

The server side defines the property here

https://github.com/elastic/elasticsearch/blob/32f46f2214808c0b2c2271bf977f2f08fe6d6fbc/server/src/main/java/org/elasticsearch/script/ScriptCacheStats.java#L108